plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseTestElasticsearchConnectorTest.java

```diff
@@ -106,6 +155,68 @@ public void testDescribeTable()
         assertEquals(actualColumns, expectedColumns);
     }
 
+    @Test
+    @Override
+    public void testPredicateReflectedInExplain()
+    {
+        // The format of the string representation of what gets shown in the table scan is connector-specific
+        // and there's no requirement that the conform to a specific shape or contain certain keywords.
```

Comment:
While this is true I think as an end-user it makes sense to make them consistent where possible. After all it's part  of the explain output.

WDYT @findepi @martint ?
We can address this in a follow-up if needed.

Reply:
Yes, consistency is good to have. 
The problem is that ES TH doesn't reveal associated `constraint` at all. If this gets left for a follow up, let's have a TODO with an issue